messages.xml

```diff
@@ -130,5 +130,8 @@ Exception: %(error)s</field>
 
 "%(sequence)s"</field>
         </record>
+        <record model="ir.message" id="msg_table_sql_error">
+            <field name="text">Computation of "%(table)s" failed:</field>
```

Comment:
Note, however that with my other comment we could remove these changes and just handle the rollback as explained.

Reply:
> because it allows the translator to put the message at other places, not only the end.

gettext() require the "transaction" is alive. When do try/except in this cas, the transaction do "die" (rollback) and gettext() is not available.